adis_tools/schemas/qes_230310.xsd

```diff
@@ -22,7 +22,7 @@ datecode 220603
       <element type="qes:outputType" name="output" minOccurs="0" />
       <element type="qes:cpstatusType" name="STATUS" minOccurs="0"/>
       <element type="qes:cptimestepsType" name="TIMESTEPS" minOccurs="0"/>
-      <element type="qes:statusType" name="exit_status"  minOccurs="0" maxOccurs="1"/>
+      <element type="qes:statusType" name="status"  minOccurs="0" maxOccurs="1"/>
```

Comment:
I had to adjust quite a few things with the xml format. Maybe something is different in the xmlschema version I use?

Reply:
ofc now some tests fail... 🙄

Reply:
I removed the modifications to the parser in https://github.com/materialdigital/ADIS2023/pull/19 and everything works fine. In addition, when I execute the notebook here locally I also get an `xml` related `Reason: Unexpected child with tag 'fcp' at position 18.` error, but the notebook execution does not fail. It seems like `jobflow` is converting the error to a warning and continues executing the remaining cells of the notebook. You can test it with [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/QuantumChemist/ADIS2023/HEAD)

Reply:
Did you test it locally with a different quantum espresso version? In the repository we currently use `qe=7.2`.

Reply:
I think it's the qe version as I used qe 6.7. Thank you for looking into this and fixing this issue 😄 
I will merge your commits back into mine to have passing tests in my repo.

Reply:
I also realized that I overlooked the environment.yml file 🙈